projects/media-viewer/src/lib/styles/media.scss

```diff
@@ -350,3 +350,15 @@
     display: none;
   }
 }
+
+@media all and (max-width: 1000px) {
+  .hidden1000px, .hidden1000px * {
```

Comment:
why 1000px?

Reply:
it turns out to be the right width for the annotation buttons to disappear